epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/madgraph/iolibs/template_files/gpu/cudacpp.mk

```diff
@@ -233,6 +233,8 @@ override OMPFLAGS = -fopenmp
 else ifneq ($(shell $(CXX) --version | egrep '^(Apple clang)'),)
 override OMPFLAGS = # AV disable OpenMP MT on Apple clang (builds fail in the CI #578)
 ###override OMPFLAGS = -fopenmp # OM reenable OpenMP MT on Apple clang? (AV Oct 2023: this still fails in the CI)
+else ifneq ($(UNAME), 'Darwin')
```

Comment:
@oliviermattelaer I just realised there are two bugs here
- It should be ifeq not ifneq
- It should be UNAME_S not UNAME
That is it should read `else ifeq ($(UNAME_S), 'Darwin')`
The way it is done now disables OpenMP on Mac AND ALSO on Linux (except icpx and clang)
I will fix this

Reply:
Ouf. There were three errors. No quotes needed. It should read `else ifeq ($(UNAME_S),Darwin)`. Fixing this again